package.json

```diff
@@ -31,5 +31,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
```

Comment:
I think object spreading is already natively supported as of ES6, so I'm not sure that we need this, unless there's something specific in this plugin you wanted to use

https://codeburst.io/a-simple-guide-to-destructuring-and-es6-spread-operator-e02212af5831

Reply:
I was getting an error when I tried it locally, but I'll revert and try again, maybe it was a problem with how I was doing it. This is why I put it in its own commit :P